internal/pkg/cli/interfaces.go

```diff
@@ -247,6 +251,23 @@ type wsSvcDirReader interface {
 	CopilotDirPath() (string, error)
 }
 
+type wsJobLister interface {
+	JobNames() ([]string, error)
+}
+
+type wsJobReader interface {
+	wsJobLister
+}
+
+type wsJobDirReader interface {
+	wsJobReader
+}
+
+type JobDirManifestWriter interface {
```

Comment:
I don't see those interface is used anywhere. Why do we want to add them?

Reply:
You're right! Took them out. Thanks!